airflow/providers/databricks/hooks/databricks.py

```diff
@@ -52,6 +52,8 @@
 
 RUN_LIFE_CYCLE_STATES = ['PENDING', 'RUNNING', 'TERMINATING', 'TERMINATED', 'SKIPPED', 'INTERNAL_ERROR']
 
+LIST_ZONES_ENDPOINT = ('GET', 'api/2.0/clusters/list-zones')
```

Comment:
That was not very good choice as it's only available on AWS, and will fail on Azure & GCP databricks